development/extensions/tutorial_php_events_and_extending_extensions.rst

```diff
@@ -231,6 +238,19 @@ contained in the array keys):
       'ext/acme/demo/images/image2.png' => 'demo',
   );
 
+.. note::
+  The method ``find_from_extension`` used above will only search in that specific
+  extension. If you want to search for images in all extensions, you use ``suffix``,
+  ``directory`` and ``find`` instead:
+
+  .. code-block:: php
+
+    $finder = $extension_manager->get_finder();
+
+    $images = $finder
+        ->find('.png')
+        ->directory('/images')
+        ->find($phpbb_root_path . 'ext/');
```

Comment:
Actually find() doesn't take any argument of this kind. Only $cache and $is_dir ($is_dir is to returns files and directories)